lib/pdf_fill/filler.rb

```diff
@@ -6,14 +6,14 @@ module PdfFill
   module Filler
     module_function
 
-    PDF_FORMS = PdfForms.new('pdftk')
+    PDF_FORMS = PdfForms.new(Settings.binaries.pdftk)
     FORM_CLASSES = {
       '21P-527EZ' => PdfFill::Forms::VA21P527EZ
     }.freeze
 
     def combine_extras(old_file_path, extras_generator)
       if extras_generator.text?
-        file_path = "tmp/pdfs/form_#{Time.zone.now}_final.pdf"
+        file_path = "#{old_file_path.gsub('.pdf', '')}_final.pdf"
```

Comment:
`old_file_path` is `tmp/pdfs//#{saved_claim.form_id}_#{saved_claim.id}.pdf"`, where saved_claim is an instance of `SavedClaim`. As long as primary key ids are unique, we're fine and would only ever potentially overwrite ourselves.